### ISSUE TITLE
Name of current note in Zim-window`s title bar

### ISSUE THREAD

**Issue by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1339]
When the name of a note is long enough and its beginning is similar to the name of another note, then displaying the name of the note in the tree does not help to understand which particular note the user is editing.

Is it possible to display the full name of the current note in the title bar of the Zim window after the name of the notebook (or elsewhere)?

**Comment by fdt-dev** [comment]
It's indeed a problem that names of long notes will not be fully displayed in the tree. This is one of the reasons why I opened this issue here #1171 
Unfortunately however I had to modify the code myself in order to make it happen. Titlebar would be an option, too, but then the issue remains that it is difficult to tell which entry in the tree one should click. I'd rather have the horizontal scrollbar in the tree view. That is much more helpful, I think anyway.

**Comment by Nick222** [comment]
It seems to me that these are different options:
- horizontal scroll bar for full slow access to the note name;
- window title - for a quick check - what the user is working with.

In the title of the window, you can put the beginning of the name of the note - how much will fit.
In any case, it will be larger and more informative than in the note tree.

**Comment by pfalcon** [comment]
> Unfortunately however I had to modify the code myself in order to make it happen.

It's great fortune and luck that we live in a world where open-source has become a norm, and Zim specifically written in a language which makes it very easy for users to make any changes (Python). Enjoy, and don't forget to post the patch to let others benefit from it too (just like you benefit from Zim had been posted in the first place.)

**Comment by fdt-dev** [comment]
There is no patch as such, because I got the feeling no one was really interested in it. If you take the time to read the issue #1171 as mentioned you will find this line in there:
remove line 391 in the `file __init__.py` in the folder `plugins/pageindex`

That's the patch. Then you've got a horizontal scrollbar for the tree view.

And yes, you're right: it's great that this is open source software. It really is and I have stated before that I really love Zim Wiki.
